Review Assignment/src/assets/scripts/common.js

```diff
@@ -0,0 +1,34 @@
+window.common = function() {
```

Comment:
if it is parent file for all JS. then you can name is app.js or index.js
if it is helper file for all JS then name it to helper.js